src/object/srv_ec_aggregate.c

```diff
@@ -2521,8 +2521,14 @@ cont_ec_aggregate_cb(struct ds_cont_child *cont, daos_epoch_range_t *epr,
 		ec_agg_param->ap_agg_entry.ae_obj_hdl = DAOS_HDL_INVAL;
 	}
 
-	if (ec_agg_param->ap_obj_skipped)
-		D_ERROR("with skipped obj during aggregation.\n");
+	if (ec_agg_param->ap_obj_skipped) {
+		D_DEBUG(DB_EPC, "with skipped obj during aggregation.\n");
+		/* There is rebuild going no, and we can proceed EC aggregate boundary,
```

Comment:
just to confirm that when ap_obj_skipped is set, it only means all the parity shard is in rebuilding.
If some data shards or partial parity shard is in rebuilding, then the ap_obj_skipped flag will not be set.